README.md

```diff
@@ -53,4 +55,34 @@ look like this:
     |-- <QBIC sample code>.fastq.gz.sha256sum
 ```
 
+### HLA Typing data
+**Responsible dropbox:**
+[QBiC-register-hlatyping-dropbox](drop-boxes/register-hlatyping-dropbox)
+
+**Resulting data model in openBIS**  
+Q_TEST_SAMPLE -> Q_NGS_HLATYPING (with sample code) -> DataSet (directory
+with files contained)
+
+or
+
+Q_TEST_SAMPLE -> Q_NGS_SINGLE_SAMPLE_RUN (provided sample code) -> Q_NGS_HLATYPING -> DataSet (directory
+with files contained)
+
+Example sample ids are:
+QABCD001AE (Analyte, Q_TEST_SAMPLE)  
+HLA1QABCD001AE (HLA-Typing result, Q_NGS_HLATYPING) for HLA MHC class I
+or
+HLA2QABCD001AE (HLA-Typing result, Q_NGS_HLATYPING) for HLA MHC class II
+
+
+**Description**  
+For HLA typing data in VCF format, the file structure
+needs to look like this:
+
+```
+<QBIC sample code> // Directory
+    |-- <QBIC sample code>.txt
+    |-- <QBIC sample code>.txt.sha256sum
+```
```

Comment:
is it really `txt` and not `vcf`?

Reply:
@wow-such-code told me its txt, I also thought its vcf

Reply:
It's either .txt or .alleles, see:
https://github.com/qbicsoftware/etl-scripts/blob/documentation/register-hlatyping-drop-box/drop-boxes/register-hlatyping-dropbox/register-hlatyping.py#L114

This is really only a very small text file containing the typing results, e.g. A*29:02,A*31:01...

Reply:
There is however a wrong comment at the top script that was copied over from the vcf dropbox it seems...